README.md

```diff
@@ -96,6 +96,22 @@ Install the JavaScript with:
    react-native link react-native-svg
    ```
 
+##### React Native Web
+
+`react-native-svg` has a compatibility layer for the web. This compatibility layer is exposed via `.web.js` files.
```

Comment:
this seems like it should be documented somewhere outside of react-native-svg because it applies to basically every react-native library that supports web

Reply:
Do you know what that other resource might be? I'm inclined to agree, but I'm not familiar with this ecosystem.

Reply:
Thanks for the review by the way, @brentvatne 😄

Reply:
Could be a medium blog or something. But please elaborate a bit further on how to get react-native-web compatibility with a packages such as this one. To me "exposed via 'web.js' files doesn't clarify how to make them play nice together...